static/js/containers/CollectionDetailPage.js

```diff
@@ -192,13 +192,15 @@ export class CollectionDetailPage extends React.Component<*, void> {
     )
   }
 
-  renderEmptyVideoMessage() {
+  renderEmptyVideoMessage(isAdmin: boolean) {
+    let message = "There are no public videos available for viewing."
```

Comment:
We've confused some users who don't realize that they need to login. Can we add something like this (not sure if this will work with react) 
```suggestion
    let message = "There are no public videos available for viewing. Please <a href="/login">login</a> to view private videos"
```

Ideally that login link would direct them back to the current page after logging in.

Reply:
Might be useful to make the URL `/login/?next=/collections/<collection_key>` so the user is redirected back to the collection after logging in.